ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -722,7 +728,7 @@ class VariantGenes extends React.PureComponent {
         {!mainGeneId && (
           <div>
             {[...GENE_DISEASE_DETAIL_SECTIONS, ...GENE_DETAIL_SECTIONS].map(
-              ({ showDetails, detailsDisplay, ...sectionConfig }) => {
+              ({ showDetails, detailsDisplay, expandedDisplay, expandedLabel, ...sectionConfig }) => {
```

Comment:
Add `expandedDisplay` and `expandedLabel` to avoid passing them to GeneDetailSection through ...sectionConfig and get rid of an unexpected parameter warning.